modules/integration_azure-mysql/conf/01-cpu.yaml

```diff
@@ -0,0 +1,22 @@
+module: Azure MySQL
+name: "CPU usage"
```

Comment:
if you want you can define id like
`id: cpu`
will make variables naming shorter and "usage" is not really useful for id.
can be applied to other detectors too